fizzbuzz_yamaguchi.py

```diff
@@ -0,0 +1,9 @@
+for i in range(1, 101):
+    if i % 3 == 0 and i % 5 == 0:
+        print('FizzBuzz')
```

Comment:
I think this should be "Fizz buzz" (two words).
https://en.wikipedia.org/wiki/Fizz_buzz